libs/sdk-ui-dashboard/src/model/store/meta/metaSelectors.ts

```diff
@@ -362,6 +363,30 @@ const selectPersistedDashboardLayout = createSelector(selectSelf, (state) => {
     return state.persistedDashboard?.layout;
 });
 
+/**
+ * Selects persisted attribute filter configs - that is the attribute filter configs array object that was used to initialize the rest
+ * of the dashboard state of the dashboard component during the initial load of the dashboard.
+ *
+ * Note that this may be undefined when the dashboard component works with a dashboard that has not yet
+ * been persisted (typically newly created dashboard being edited).
+ */
+const selectPersistedDashboardAttributeFilterConfigs = createSelector(selectSelf, (state) => {
+    return state.persistedDashboard?.attributeFilterConfigs || [];
+});
+
+/**
+ * Selects a boolean indication if he dashboard has any changes to the dashboard filter compared to the persisted version (if any)
+ *
+ * @internal
```

Comment:
please consider to remove if this is unnecessary!

Reply:
Removed the @internal. Thanks!